src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -40,6 +44,7 @@ const StyledCheckBoxContainer = styled.label`
   ${props => props.disabled && disabledStyle}
   ${props => !props.disabled && 'cursor: pointer;'}
   ${props => props.theme.checkBox.hover.border.color && hoverStyle}
+  ${props => props.theme.checkBox.hover.background.color && hoverStyle}
```

Comment:
This line would break if the user won't have `props.theme.checkBox.hover.border.color` defined.